era5_cloud/README.md

```diff
@@ -0,0 +1,22 @@
+# Cloud data, monthly means, 2010
+
+This artifact repackages data coming from [ERA5 reanalysis](https://cds.climate.copernicus.eu/datasets/reanalysis-era5-pressure-levels-monthly-means?tab=overview) 
+and contains monthly averaged cloud properties in 2010.
+
+The input file is defined in pressure coordinates. We convert this into altitude over mean-sea level
+using $P = P^*exp(-z / H)$ with scale height $H$. We assume $P^* = 1e5$ (Pa) and $H = 7000$ (m). The output 
+is a NetCDF file that contains cloud fraction, cloud liquid water content (kg/kg), and cloud ice water content (kg/kg) 
+defined on a lon-lat-z-time grid.
+
+## Usage
+To recreate the artifact:
+1. Create a python virtual environment
+2. Activate the new virtual env
+3. In the same terminal run `pip install -r requirements.txt`
+4. In the same terminal run `julia --project create_artifact.jl`
+
+## Requirements
+- Python >= 3
+
+## Reference
+Hersbach, H., Bell, B., Berrisford, P., Biavati, G., Horányi, A., Muñoz Sabater, J., Nicolas, J., Peubey, C., Radu, R., Rozum, I., Schepers, D., Simmons, A., Soci, C., Dee, D., Thépaut, J-N. (2023): ERA5 monthly averaged data on pressure levels from 1940 to present. Copernicus Climate Change Service (C3S) Climate Data Store (CDS), DOI: 10.24381/cds.6860a573 (Accessed on 08-Oct-2024)
```

Comment:
I think there's extra licenses with ERA5 that we need to add here.

Reply:
The license information is here (in the right panel): https://cds.climate.copernicus.eu/datasets/reanalysis-era5-single-levels-monthly-means?tab=download. What should we include here?

Reply:
We should at least include a link to the original license

Reply:
I added a link to the licence pdf